### DISCUSSION TITLE
How to add insecureSkipTLSVerify=true for S3 bucket connection in K8up backup.yaml resource?

### DISCUSSION THREAD

**Discussion by vistasunil** [k8up-io/k8up#881] [Q&A]
Hello Team,

Provided [backup.yaml](https://github.com/k8up-io/k8up/blob/master/docs/modules/ROOT/examples/tutorial/backup.yaml) example file including K8up documentation, doesn't mentions anything about enabling `insecureSkipTLSVerify=true` flag for connecting to S3 buckets:

I am blocked due to this. Is there any way we can achieve the same? 
If anyone knows, then kindly share the details.

Thanks & Regards,
Sunil Kumar

**Comment by Kidswiss** [comment]
Hi @vistasunil 

This is currently not supported. There's a feature request to pass more/arbitrary flags to restic here: https://github.com/k8up-io/k8up/issues/792
